src/Client.php

```diff
@@ -69,6 +69,52 @@ public function get($uri)
         return $parsedResponse;
     }
 
+    public function postFile($realFilePath, $uploadFileName = null)
+    {
+        if (!$uploadFileName) {
+            // No specified filename, use the uploaded one
+            $uploadFileName = pathinfo($realFilePath)['basename'];
+        }
+        $me = $this->get('me');
```

Comment:
May be this API request to /me should be passed as an optional method parameter in order to allow saving some API calls ?